src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -528,7 +528,7 @@ namespace AppInstaller::Repository
             else
             {
                 AICLI_LOG(Repo, Info, << "Default source requested, multiple sources available, creating aggregated source.");
-                auto aggregatedSource = std::make_shared<AggregatedSource>();
+                auto aggregatedSource = std::make_shared<AggregatedSource>("*DefaultSource");

```

Comment:
>DefaultSource [](start = 77, length = 13)

"*DefaultAggregatedSource" to better denote it's a multiple default source?

Reply:
I don't think that is necessary.

---
In reply to: [492435549](https://github.com/microsoft/winget-cli/pull/575#discussion_r492435549) [](ancestors = 492435549)